src/main/java/sa/assertj/maps/ContainsExactlyInAnyOrderEntriesOf.java

```diff
@@ -0,0 +1,41 @@
+package sa.assertj.maps;
+
+import sa.assertj.Experiment;
+import sa.assertj.Util;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ContainsExactlyInAnyOrderEntriesOf extends Experiment {
+
+    static DataProvider provider = (size, numOfSamples) -> {
+        Object[][] result = new Object[numOfSamples][];
+        for (int s=0; s < numOfSamples; s++) {
+            Map<String, String> actual = new LinkedHashMap<>();
+            Map<String, String> expected = new LinkedHashMap<>();
+            for (int i=0; i < size; i++) {
+                String key = Util.randomString(10);
+
+                if (actual.containsKey(key)) {
+                    i--;
+                    continue;
+                }
+
+                String value = Util.randomString(10);
+
+                actual.put(key, value);
+                expected.put(key, value);
```

Comment:
Same here. This again should be unsorted imo cause otherwise it's skewed. At least one of them should be shuffled. If you wanna keep the same type, then just take a set of the keys at the end, shuffle them into a sorted boi and create the expected map from the actual one by searching out the corresponding values.

Reply:
I think I did this? Already don't remember